test/unit_tests/location_service/set_distance_between_user_and_games_test.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:niira/models/location.dart';
```

Comment:
just a comment on the file name - in the same way that it's helpful to have the widget tests named something like "big_widget_test.dart" I reckon it's helpful to have unit tests named as "thing_test.dart" (no need for "unit" in the name, just a convention i think).  I definitely could be wrong, might be worth fact checking me.  So in this case I would suggest maybe "location_service_test.dart" maybe under "test/unit_tests/services/"... not sure how much this is a convention and how much a style choice, so just a suggestion in case it's helpful and please ignore me if not 🙂

Reply:
thanks nick!!